cerberus/models/booking.py

```diff
@@ -57,38 +69,60 @@ def round_date_time(dt: datetime) -> datetime:
 
         return make_aware(dt)
 
-    def _valid_dates(self) -> bool:
-        return self.end > self.start
-
     def get_overlapping(self) -> QuerySet[Self]:
         start = Q(start__lt=self.start, end__gt=self.start)
         end = Q(start__lt=self.end, end__gt=self.end)
         equal = Q(start=self.start, end=self.end)
+        contains = Q(start__gt=self.start, end__lt=self.end)
 
-        return self.__class__.objects.filter(start | end | equal).exclude(pk=self.pk)  # type: ignore
+        return self.__class__.objects.filter(start | end | equal | contains).exclude(pk=self.pk)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider simplifying the overlap check logic.

The current implementation of the `overlaps` method might be simplified by directly using Django's `__overlap` lookup in the query. This could potentially improve readability and performance.

```suggestion
def get_overlapping(self) -> QuerySet[Self]:
    return self.__class__.objects.filter(
        Q(start__lt=self.end, end__gt=self.start)
    ).exclude(pk=self.pk)
```